spyder/utils/help/tutorial.rst

```diff
@@ -761,100 +732,99 @@ Debugging
-*breakpoint* by pressing F12 on the line on which you want to stop. After
-that a red dot will be placed next to the line and you can press the ``Continue``
-button
+*breakpoint* by pressing F12 on the line on which you want to stop, or
+double-clicking to the left of the line number. A red dot in this position
+indicates a breakpoint; it can be removed by repeating the same procedure.
+
+After entering the Debugger, you can press the ``Continue`` button
 
 .. image:: images/debug-continue.png
      :align: center
 
-(after entering debug mode) to stop the execution at that line.
+to stop the execution at the first breakpoint.
 
 .. note::
 
-   You can also control the debugging process by issuing these commands in the
+   You can also control the debugging process by issuing these commands at the
    console prompt:
-   
-   * ``n`` to move to the Next statement. 
-   
+
+   * ``n`` to move to the Next statement.
+
    * ``s`` to Step into the current statement. If this is a function
-     call, step into that function. 
+     call, step into that function.
 
    * ``r`` to complete all statements in the current function and Return
      from that function before returning control.
 
-   * ``p`` to print values of variables, for example ``p x`` will print the
-     value of the variable ``x``.
+Inside the debugger, you can also interactively execute many of the statements
+you can normally, including assigning and modifying variables, defining and
+calling functions, setting new breakpoints, and more.
 
-At the debugger prompt, you can also *change* values of variables. For
-example, to modify a variable ``x`` at the IPython debugger prompt, you can say
-``ipdb > x = 42`` and the debugger will carry on with ``x`` being bound to ``42``.
-You can also call functions, and do many others things. Try this example::
+For example, enter the following code into a new or temporary file::
 
    def demo(x):
        for i in range(5):
-           print("i={}, x={}".format(i, x))
+           print("i = {}, x = {}".format(i, x))
            x = x + 1
-        
+
    demo(0)
 
 If we execute this (``Run > Run``), we should see the output::
 
-	i=0, x=0
-	i=1, x=1
-	i=2, x=2
-	i=3, x=3
-	i=4, x=4
-	
-Now execute this using the debugger (``Debug > Debug``), press the
-``Step button`` until the highlighted line reaches the ``demo(0)``
+	i = 0, x = 0
+	i = 1, x = 1
+	i = 2, x = 2
+	i = 3, x = 3
+	i = 4, x = 4
+
+Now run this using the debugger (``Debug > Debug``), press the
+``Step`` button until the highlighted line reaches the ``demo(0)``
 function call, then press the ``Step into`` to inspect this function.
-Keep pressing the ``Step button`` to execute the next lines. Then,
-modify ``x`` by typing ``x=10`` in the debugger prompt. You see x
-changing in the Variable Explorer. You should also see ``x`` changing
+Keep pressing the ``Step`` button to execute the next lines. Then,
+modify ``x`` by typing ``x = 10`` in the debugger prompt. You should see x
```

Comment:
"x" at the end of the line needs to have double backpacks on either side for correct rendering: "``x``"

Reply:
Fixed!